src/main/resources/assets/fabric-fluid-volume-v1/lang/en_us.json

```diff
@@ -0,0 +1,6 @@
+{
+  "and": " and ",
```

Comment:
should use a different translation key here

Reply:
what should it be

Reply:
`text.fluid.and`